gitlab.yml

```diff
@@ -0,0 +1,26 @@
+provider:
+  name: faas
+  gateway: http://127.0.0.1:8080
+
+functions:
+  system-gitlab-event:
+    lang: go
+    handler: ./gitlab-event
+    image: martindekov/gitlab-event:0.0.1
+    labels:
+      openfaas-cloud: "1"
+      role: openfaas-system
+    environment:
+      content_type: text/plain
+      validate_customers: true
+      validate_token: false
+      write_debug: true
+      read_debug: true
+      installation_tag: "openfaas-cloud"
```

Comment:
Make sure this defaults in the code for when it's commented out or deleted by mistake.

Reply:
Yeah currently it is